### PR TITLE
feat(modules/cloud-logs): introduce cloud logs module for s3 onboarding

### PR DESCRIPTION
**Pull request by SKosier**

### CODE REVIEW

**Review by mancia93 — APPROVED** [review]
LGTM 🚀

File: modules/integrations/cloud-logs/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+| [sysdig_secure_cloud_auth_account_component.aws_cloud_logs](https://registry.terraform.io/providers/sysdiglabs/sysdig/latest/docs/resources/secure_cloud_auth_account_component) | resource |
+
+## Inputs
+
+| Name                                                               | Description                                                                                                                                    | Type | Default                                                     | Required |
+|--------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|------|-------------------------------------------------------------|:--------:|
+| <a name="input_sysdig_secure_account_id"></a> [sysdig\_secure\_account\_id](#input\_sysdig\_secure\_account\_id) | (Required) ID of the Sysdig Cloud Account to enable Event Bridge integration for (incase of organization, ID of the Sysdig management account) | `string` | n/a | yes |
```
Comment (legobrick) [inline modules/integrations/cloud-logs/README.md:45]:
```suggestion
| <a name="input_sysdig_secure_account_id"></a> [sysdig\_secure\_account\_id](#input\_sysdig\_secure\_account\_id) | (Required) ID of the Sysdig Cloud Account to enable Cloud Logs integration for (incase of organization, ID of the Sysdig management account) | `string` | n/a | yes |
```

File: modules/integrations/cloud-logs/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+| <a name="input_folder_arn"></a> [folder\_arn](#input\_folder\_arn) | (Required) The ARN of your CloudTrail Bucket Folder                                                                                            | `string` | n/a                                                         |   yes    |
+| <a name="input_bucket_arn"></a> [bucket\_arn](#input\_bucket\_arn) | (Required) The ARN of your s3 bucket associated with your Cloudtrail trail                                                                     | `string` | n/a                                                         |   yes    |
```
Comment (legobrick) [inline modules/integrations/cloud-logs/README.md:47]:
```suggestion
| <a name="input_bucket_arn"></a> [bucket\_arn](#input\_bucket\_arn) | (Required) The ARN of your S3 bucket associated with your CloudTrail Trail                                                                     | `string` | n/a                                                         |   yes    |
```

File: modules/integrations/cloud-logs/main.tf [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+    resources = [
+      var.bucket_arn,
+      "${var.bucket_arn}/*"
+    ]
+  }
+}
+
+#-----------------------------------------------------------------------------------------------------------------------------------------
+# Call Sysdig Backend to add the event-bridge integration to the Sysdig Cloud Account
```
Comment (legobrick) [inline modules/integrations/cloud-logs/main.tf:106]:
```suggestion
# Call Sysdig Backend to add the cloud logs integration to the Sysdig Cloud Account
```

File: modules/integrations/cloud-logs/README.md (hunk 2 above)
Comment (legobrick) [inline modules/integrations/cloud-logs/README.md:47]: Should we ditch this as we move to modular, in favor of the folder arn? The bucket arn can be derived from that...

File: modules/integrations/cloud-logs/main.tf [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  type                       = "COMPONENT_CLOUD_LOGS"
+  instance                   = "secure-runtime"
+  version                    = "v0.1.0"
+  cloud_logs_metadata = jsonencode({
+    aws = {
+      cloudtrailS3Bucket = {
+        folder_arn    = var.folder_arn
+        role_name     = local.role_name
+        external_id   = data.sysdig_secure_tenant_external_id.external_id.external_id
+        region        = var.region
```
Comment (legobrick) [inline modules/integrations/cloud-logs/main.tf:122]: Before releasing this, let's remove the region, so we avoid "polluting" some other setups and come up with something clean. Ok to keep this on a branch for internal testing for now, but let's remove it before releasing, along with the variable etc.

**Review by legobrick — CHANGES_REQUESTED** [review]

File: modules/integrations/cloud-logs/main.tf (hunk 4 above)
Comment (SKosier) [reply modules/integrations/cloud-logs/main.tf:122]: Agree. Will open a PR that extends this one and removes the region. So we can prepare backend changes in the meantime.

**Review by legobrick — APPROVED** [review]

**Pull request merged**